src/fides/api/ops/schemas/application_config.py

```diff
@@ -57,7 +60,8 @@ def validate_notification_service_type(cls, value: str) -> Optional[str]:
 
 
 class ExecutionApplicationConfig(BaseSchema):
-    subject_identity_verification_required: bool
+    subject_identity_verification_required: Optional[bool]
+    require_manual_request_approval: Optional[bool]
```

Comment:
Instead of `Optional` could these have a default value (`False` I assume) so you don't later have to check for both `None` and either `True` or `False`?

```suggestion
    subject_identity_verification_required: bool = False
    require_manual_request_approval: bool = False
```

Reply:
hmmm, well here a default value actually means something different than none/null. if the value is set here via API (even by default assignment), it's going to override what's been set via non-API config (i.e. env var or .toml), and i don't think that's the behavior we want. does that make sense?

Reply:
This is referring to the value in `src/fides/api/ops/schemas/application_config.py` and you want this to determine the default instead? If yes, this make sense.

```py
        subject_identity_verification_required: bool = Field(
        default=False,
        description="Whether privacy requests require user identity verification.",
    )
```

Reply:
Both of those are correct @sanders41 

Config values must use `Field` (or it'll trigger a CI failure, after my PR is merged) and `Optional` fields are also no longer accepted (also a CI failure). This is due to the metadata here being used to generate valid `toml`

Reply:
right - i think this all adds up fine, but i want to make sure we're all on the same page.

here my code is effectively an API schema/model - it's _not_ application config, even though it will eventually be used alongside/reconciled with the 'proper' application config. i want to keep this field as optional in the API schema, and i do _not_ want to default to a certain value, because the config proxy will take API-set properties over the corresponding config-set properties. if we were to default here in the API model, then it'd mean that default would _override_ the default set on the application config - and that's not desirable.

but i think this is perfectly in line with @ThomasLaPiana 's changes -- essentially the `subject_identity_verification_required` field in the 'proper' application config sets the default, and here in the API-level we don't provide any sort of default, we allow it to be null/None which essentially means - "nothing has been set for this property via API, instead rely on the config setting from the 'proper' application config".

Reply:
This makes sense to me. @ThomasLaPiana will this pass in your new CI changes?

Reply:
Looked into this closer and since it isn't part of the application config files it should be fine in CI.